src/cpp/session/resources/tutorial_resources/loading.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+
+   <head>
+      <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+      <title>Loading Tutorial</title>
+
+<style>
+
+/**
+ * Modified from https://loading.io/css/.
+ * Released under the CC0 License.
+ */
+
+.lds-spinner {
+  display: inline-block;
+  position: relative;
+  width: 40px;
+  height: 28px;
+}
+
+.lds-spinner div {
+  transform-origin: 20px 20px;
+  animation: lds-spinner 1.2s linear infinite;
+}
+
+.lds-spinner div:after {
+  content: " ";
+  display: block;
+  position: absolute;
+  top: 3px;
+  left: 18px;
+  width: 3px;
+  height: 8px;
+  border-radius: 20%;
+}
+
+.rstudio-themes-light-menus .lds-spinner div:after {
+   background: rgb(128, 128, 128);
+}
+
+.rstudio-themes-dark-menus .lds-spinner div:after {
+   background: white;
+}
+
+.lds-spinner div:nth-child(1) {
+  transform: rotate(0deg);
+  animation-delay: -1.1s;
+}
+.lds-spinner div:nth-child(2) {
+  transform: rotate(30deg);
+  animation-delay: -1s;
+}
+.lds-spinner div:nth-child(3) {
+  transform: rotate(60deg);
+  animation-delay: -0.9s;
+}
+.lds-spinner div:nth-child(4) {
+  transform: rotate(90deg);
+  animation-delay: -0.8s;
+}
+.lds-spinner div:nth-child(5) {
+  transform: rotate(120deg);
+  animation-delay: -0.7s;
+}
+.lds-spinner div:nth-child(6) {
+  transform: rotate(150deg);
+  animation-delay: -0.6s;
+}
+.lds-spinner div:nth-child(7) {
+  transform: rotate(180deg);
+  animation-delay: -0.5s;
+}
+.lds-spinner div:nth-child(8) {
+  transform: rotate(210deg);
+  animation-delay: -0.4s;
+}
+.lds-spinner div:nth-child(9) {
+  transform: rotate(240deg);
+  animation-delay: -0.3s;
+}
+.lds-spinner div:nth-child(10) {
+  transform: rotate(270deg);
+  animation-delay: -0.2s;
+}
+.lds-spinner div:nth-child(11) {
+  transform: rotate(300deg);
+  animation-delay: -0.1s;
+}
+.lds-spinner div:nth-child(12) {
+  transform: rotate(330deg);
+  animation-delay: 0s;
+}
+
+@keyframes lds-spinner {
+  0%   { opacity: 1; }
+  100% { opacity: 0; }
+}
+
+</style>
+
+   </head>
+
+   <body class="rstudio-tutorials">
+      <div>
+
+         <div role="img" class="lds-spinner">
```

Comment:
Is there a more appropriate ARIA role to suggest that the "stuff" in here isn't relevant to a screen reader? (Since these are just divs that form an animation)